docs/src/prototypes/dropdowns/AsyncDropdownSearch.tsx

```diff
@@ -68,33 +68,26 @@ class AsyncDropdownSearch extends React.Component<{}, SearchPageState> {
     const { items, loading, searchQuery, value } = this.state
 
     return (
-      <div style={{ margin: 20 }}>
-        <Segment>
-          <Header content="Async Dropdown Search" />
-          <p>Use the field to perform a simulated search.</p>
-        </Segment>
-
-        <Segment>
-          <Dropdown
-            fluid
-            items={items}
-            loading={loading}
-            loadingMessage={{
-              content: <Loader label="Loading..." labelPosition="end" size="larger" />,
-            }}
-            multiple
-            onSearchQueryChange={this.handleSearchQueryChange}
-            onSelectedChange={this.handleSelectedChange}
-            placeholder="Try to enter something..."
-            search
-            searchQuery={searchQuery}
-            toggleIndicator={false}
-            value={value}
-          />
-          <Divider />
-          <CodeSnippet mode="json" value={this.state} />
-        </Segment>
-      </div>
+      <>
+        <Dropdown
```

Comment:
for this experience might suggest to introduce
- comment to example's code
- maybe better additional `<p>` in between of `Dropdown` and `CodeSnippet`, or add text to `Divider`

that would explain that `CodeSnippet` part rendered is necessary to reflect state of `fetchedData` - otherwise it was a bit confusing to me an, frankly, originally I thought that this is just a docs bug :) In either case, it would be helpful to explicitly say to the client that this is aimed to represent state of _fetched_ data.

![image](https://user-images.githubusercontent.com/9024564/53235887-f2a54f80-36a3-11e9-8f98-497a58a1a4ee.png)

Reply:
also, when there is nothing found in the list, there is the following view of Dropdown produced:

![image](https://user-images.githubusercontent.com/9024564/53236133-9db60900-36a4-11e9-993e-3b93224f743f.png)

This representation (with narrow white ribbon below the search field) seems to be a bit confusing - I think that we could easily fix it with `noResultsMessage` prop:

![image](https://user-images.githubusercontent.com/9024564/53236285-f685a180-36a4-11e9-8c19-85a71565f4b7.png)

Reply:
@kuzhelov `AsyncDropdownSearch.tsx` is not part of this PR;
 this file was just moved so that all dropdown related prototypes are in one place